src/Compat/CompatForm.php

```diff
@@ -3,15 +3,33 @@
 namespace ipl\Web\Compat;
 
 use ipl\Html\Form;
+use ipl\Html\HtmlDocument;
+use ipl\Html\HtmlString;
 use ipl\I18n\Translation;
+use ipl\Web\Common\PrefixSubmitButton;
 use ipl\Web\FormDecorator\IcingaFormDecorator;
 
 class CompatForm extends Form
 {
     use Translation;
+    use PrefixSubmitButton;
 
     protected $defaultAttributes = ['class' => 'icinga-form icinga-controls'];
 
+    public function renderContent(): string
```

Comment:
Please document here that this injects a duplicate of the submit button at the start.